felix/bpf/tc/defs/defs.go

```diff
@@ -50,9 +52,13 @@ const (
 	ProgIndexV6Allowed
 	ProgIndexV6Icmp
 	ProgIndexV6Drop
+
+	ProgIndexStart = ProgIndexPolicy
```

Comment:
Where this const is used? cannot find it.

Reply:
no used, will remove :+1: